docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -383,7 +383,7 @@ Confirm Network Functionality
 
        .. code-block:: console
 
-          $ curl http://sawtooth-poet-rest-api-0:8008/peers
+          $ curl http://sawtooth-rest-api-default-2:8008/peers
```

Comment:
Thanks for fixing the container name.  However, the change from 0 to 2 is not correct.  (A name that ends with -2 is for the REST API of the third validator node, but the text says to run this command on the first validator node.) Please change 2 back to 0.